css/ampshared.css

```diff
@@ -412,13 +411,6 @@ form [submit-error] {
   display: none;
 }
 
-/**
- * amp-list error messaging container should be hidden at first.
- */
-amp-list [fetch-error] {
```

Comment:
For my own learning, do we have any place where we've written what should go in the various css files? E.g. whats the the logic behind what goes in `ampshared.css` vs. `ampdoc.css`

Reply:
ampshared.css is shared with AMP4ADS. E.g. CSS for AMP components that are not allowed in AMP4ADS should not appear in ampshared.css.

I'll add a `@fileoverview` to both files explaining this.

Reply:
Can we also add enforcement for this in a future PR? So new authors don’t accidentally commit CSS changes to the wrong place?

Reply:
Good question but not sure how to enforce this since the AMP/AMP4ADS boundary is not easily defined. 

Perhaps adding a OWNERS `notify` to wg-ads group for changes to ampshared.css?